packages/ra-core/src/controller/input/ReferenceArrayInputController.tsx

```diff
@@ -4,6 +4,7 @@ import debounce from 'lodash/debounce';
 import compose from 'recompose/compose';
 import { createSelector } from 'reselect';
 import isEqual from 'lodash/isEqual';
+import difference from 'lodash/difference'
```

Comment:
Should we use the comma at the end of line style?

Reply:
Yes, please configure your editor to apply our prettier rules

Reply:
Ah yes my bad. @djhi I noticed that `make prettier` changed a whole bunch of files that don't seem to respect those rules. Would it be ok to create a separate pull request with all those changes ?

Reply:
Please don't. Can you just apply prettier on the file you modified? If you can automate it and our make command change too many files, just run the `lint` make command and fix manually. Thanks!